File: swagger/models/update_client_service_response.py
from .base_model import BaseModel

from .client_service import ClientService

class UpdateClientServiceResponse(BaseModel):
	"""
	no description available

	Attributes:

		client_service
		Contains information about the updated service.
	"""

	input_map = {
		'ClientService': ('client_service', ClientService),
		}

	output_map = {
		'client_service': 'ClientService',
		}

		'ClientService': ('client_service', ClientService),

File: swagger/Models/UpdateClientServiceResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class UpdateClientServiceResponse extends BaseModel
{
	/**
	 * Contains information about the updated service.
	 * 
	 * @var ClientService 
	 */
	public $clientService;

	protected function getInputMap()
	{
		return [
			'ClientService' => ['clientService', ClientService::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'clientService' => ['ClientService', ClientService::class, null],
			];
	}
}
